src/app/app.component.ts

```diff
@@ -229,4 +241,23 @@ export class AppComponent implements OnInit, AfterViewInit {
     };
     head.appendChild(link);
   }
+
+  private trackIdleModal() {
+    const isIdle$ = this.authService.isUserIdle();
+    const isAuthenticated$ = this.authService.isAuthenticated();
+    isIdle$.pipe(withLatestFrom(isAuthenticated$))
+      .subscribe(([userIdle, authenticated]) => {
+        if (userIdle && authenticated) {
+          if (!this.idleModalOpen) {
+            const modalRef = this.modalService.open(IdleModalComponent);
```

Comment:
Tiny thing, could we pass [an `ariaLabelledBy`](https://ng-bootstrap.github.io/#/components/modal/api#NgbModalOptions) to this `open()` call, to add better Accessibility for this modal?  

When I scanned this modal using Deque's Axe browser plugin, it shows the modal is missing a label/title.  It looks (to me), like we could fix that quickly by adding an `ariaLabelledBy` option to this line...and have it reference the `idle-modal.header` text by adding a `id` to that `<div>` in the `idle-modal.component.html`.